common/lib/xmodule/xmodule/tabs.py

```diff
@@ -402,6 +403,13 @@ def initialize_default(course):
             CourseTab.load('progress'),
         ])
 
+        # If the feature is enabled, add sketchfab viewer tab in every course
+        # by default.
+        if settings.FEATURES.get('ENABLE_SKETCHFAB_VIEWER_TAB'):
+            course.tabs.extend([
```

Comment:
for only one item you can simply do append instead of extending.

Reply:
done